### PR TITLE
[develop] Game information is now represented in a Game object

### PR DESCRIPTION
**Pull request by SamSchlegel**

### CODE REVIEW

File: discord/client.py [hunk 1]
```diff
@@ -1444,27 +1446,25 @@ def delete_channel_permissions(self, channel, target):
...
-        :param game_id: The numeric game ID being played. None if no game is being played.
+        :param game: A Game object representing the game being played. None if no game is being played.
         :param idle: A boolean indicating if the client should go idle."""
 
         idle_since = None if idle == False else int(time.time() * 1000)
         payload = {
             'op': 3,
             'd': {
-                'game_id': game_id,
+                'game': game and game.__dict__,
```
Comment (SamSchlegel) [inline discord/client.py]: I'm thinking this should maybe be explicitly created

File: discord/game.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015 Rapptz
```
Comment (SamSchlegel) [inline discord/game.py]: Didn't add myself to this but I can if you want

**Comment by khazhyk** [comment]
@SamSchlegel this is incomplete https://github.com/SamSchlegel/discord.py/blob/develop-game-object/discord/client.py#L218

**Comment by khazhyk** [comment]
Rebased in https://github.com/Rapptz/discord.py/commit/2f61d086e4f66eec77ed53edee3f3578639d4e4e

**Pull request closed without merging**